posix.mak

```diff
@@ -139,6 +139,10 @@ endif
 ifdef ENABLE_COVERAGE
 DFLAGS  += -cov
 endif
+ifneq (,$(TZ_DATABASE_FILE_DIR))
```

Comment:
I know that not every parameter is documented, but we should still try to document new options/flags at the "readme" on top.

Reply:
For sure but is the README.md really the proper place for this option. It's too unimportant to get such a top level mention.
Maybe a new separate build helper file or something.

Reply:
Ah you talk about the readme at the top of the file. :-D
Will add some documentation.

Reply:
Oh I meant the header of this Makefile: https://github.com/dlang/phobos/blob/master/posix.mak

As an example: I recently tried to cleanup the [Makefile header of dlang.org](https://github.com/dlang/dlang.org/blob/master/posix.mak) and there's also this very informative header of [DMD](https://github.com/dlang/dmd/blob/master/src/posix.mak).

Reply:
Done